src/math/Kconfig

```diff
@@ -9,6 +9,15 @@ config CORDIC_FIXED
 	  Select this to enable sin(), cos(), asin(), acos(),
 	  and cexp() functions as 16 bit and 32 bit versions.
 
+config LUT_TRIG_FIXED
```

Comment:
We need a cleanup at some point where we have ```MATH_TRIG_``` prefix and likewise convention for all maths APIs, macros, Kconfigs etc.

Reply:
Yep, done